srcpkgs/konquest/template

```diff
@@ -0,0 +1,16 @@
+# Template file for 'konquest'
+pkgname=konquest
+version=18.12.0
+revision=1
+build_style=cmake
+makedepends="qt5-devel qt5-svg-devel kconfig-devel kcoreaddons-devel
+kcrash-devel kdbusaddons-devel kdoctools-devel kguiaddons-devel
```

Comment:
every new line on the same variable must be prefixed with a whitespace char

Reply:
Thank you, fixed